src/Cache.ts

```diff
@@ -31,4 +31,8 @@ export default class Cache<T> {
       this.cache.restore(data as T);
     }
   }
+
+  purge(): void {
+    this.cache.restore({} as T);
```

Comment:
Hmm... I'm a little worried about what kind of effects this may produce on a running app. Did you try this out with `InMemoryCache`?

I feel like this could cause the UI to update against an empty cache, which would clear out everything. And with the `write` trigger, it seems possible for this to get caught in a loop where a write triggers a purge, which triggers another fetch + write, which triggers a purge, etc.

How would you feel about just purging the storage provider if the serialized size is `> maxSize`. That would have the effect of leaving the current app session running smoothy, but then cause it to start from a fresh, empty cache on next launch. (I know it's not ideal to have a launch with no existing data, but it's not going to happen often.)

Reply:
After our discussion, I'm in favor of stopping persistence + starting w/ a fresh, empty cache instead of clearing out the Apollo cache in memory. I'll make the edits 👍